coreos/provision-coreos.sh

```diff
@@ -45,9 +45,13 @@ chmod +x docker-compose
 mkdir -p postgres
 ./docker-compose build
 
-sudo cp nanocloud.architecture.service /etc/systemd/system/nanocloud.architecture.service
+docker pull glyptodon/guacd
```

Comment:
It would be great to precise container tag to avoid mysterious build error as pulling the lastest may include unwanted change.